articles/search/search-indexer-field-mappings.md

```diff
@@ -290,4 +290,24 @@ When you retrieve the encoded key at search time, you can then use the `urlDecod
       "name" : "urlDecode"
     }
   }]
- ```
\ No newline at end of file
+ ```
+ 
+ ### fixedLengthEncode function
```

Comment:
Add anchor before this like other mapping functions

e.g. `<a name="fixedLengthEncode"></a>`